apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -350,3 +352,6 @@ state_gas_cost(Tag, {delta, TTL}) ->
     aec_governance_utils:state_gas_cost(
       aec_governance:state_gas_cost_per_block(Tag),
       TTL).
+
+to_sign({W1, W2}) ->
+    <<W1:256, W2:256>>.
```

Comment:
I see that, because sign_t is `{tuple, [word, word]}` (in this PR) and [`aest_data:from_binary` decodes `word` as 256 bits unsigned](https://github.com/aeternity/epoch/blob/cbb26b2f8588014efc719bb8f9337f52dba62175/apps/aesophia/src/aeso_data.erl#L175) so not loss of data. Ok.